src/Uno.UI/UI/Xaml/Media/VisualTreeHelper.cs

```diff
@@ -659,9 +659,14 @@ internal static void ClearChildren(UIElement view)
 				}
 			}
 
+#if __SKIA__
+			// We still need to check the renderingBounds in case the element is clipped.
```

Comment:
Is this a TODO before we merge the PR ?

Reply:
No, it's just a (probably unnecessary) note for the next line. I adjusted the comments.